packages/@react-aria/tabs/docs/useTab.mdx

```diff
@@ -0,0 +1,162 @@
+
+```tsx example export=true
+import {Item} from '@react-stately/collections';
+import {useFocus} from '@react-aria/interactions';
+import {mergeProps} from '@react-aria/utils';
+
+function Tabs(props) {
+  let state = useTabListState(props);
+  let tablistRef = React.useRef();
+  let tabpanelRef = React.useRef();
+  let {tabListProps} = useTabList(props, state, tablistRef);
+  let {tabPanelProps} = useTabPanel(props, state, tabpanelRef);
+  return (
+    <div style={{height: '150px'}}>
+      <div {...tabListProps} ref={tablistRef}>
+        <div style={{display: 'flex', borderBottom: '1px solid grey'}}>
+          {[...state.collection].map((item) => (
+            <Tab key={item.key} item={item} state={state} />
+          ))}
+        </div>
+      </div>
+      <div {...tabPanelProps} ref={tabpanelRef} style={{padding: '10px'}}>
+        {state.selectedItem && state.selectedItem.props.children}
+      </div>
+    </div>
+  );
+}
+function Tab({item, state}) {
+  let {key, rendered} = item;
+  let ref = React.useRef();
+  let {tabProps} = useTab({key}, state, ref);
+  let isSelected = state.selectedKey === key;
+  let isDisabled = state.disabledKeys.has(key);
+  let tabStyle = {
+    padding: '10px',
+    borderBottom: isSelected ? '3px solid orange' : undefined,
+    opacity: isDisabled ? '0.5' : undefined
+  };
+  return (
+    <div {...tabProps} ref={ref} style={tabStyle}>
+      {item.rendered}
+    </div>
+  );
+}
+
+<Tabs aria-label="History of Ancient Rome" disabledKeys={['Emp']}>
+  <Item key="FoR" title="Founding of Rome">Arma virumque cano, Troiae qui primus ab oris.</Item>
+  <Item key="MaR" title="Monarchy and Republic">Senatus Populusque Romanus.</Item>
+  <Item key="Emp" title="Empire">Alea jacta est.</Item>
+</Tabs>
+```
+
+## With focusable content
+
+When the tab panel doesn't contain focusable content, the entire panel is given a `tabIndex=0` so that the content can be navigated to. When the tab panel contains focusable content, such as a textfield, then the tab index is omitted because the content itself can receive focus.
+
+Using the same example `Tabs` component from above. Try navigating from the tabs to the content for each panel using the keyboard.
+
+```tsx example
+import {TextField} from '@react-spectrum/textfield';
+import {Content} from '@react-spectrum/view';
+
+<Tabs aria-label="Notes app">
+  <Item key="item1" title="Jane Doe">
+    <Content><TextField label="Leave a note for Jane" /></Content>
+  </Item>
+  <Item key="item2" title="John Doe">Senatus Populusque Romanus.</Item>
+  <Item key="item3" title="Joe Bloggs">Alea jacta est.</Item>
+</Tabs>
+```
+
+## Internationalization
+
+useTab handles some aspects of internationalization automatically. For example, keyboard navigation left right is automatically swapped for RTL languages. You are responsible for localizing all tab labels and content.
```

Comment:
What other i18n things dose useTab do?
I also saw `direction` from useLocale() in useTabList not useTab.

Reply:
maybe I should start the sentence with useTabList, but was going for the overall class of hooks in which I am missing an s on the end of that
mostly it's just confusing to refer to the three hooks together

Reply:
opting to change it to `These hooks handle some aspects`